docs/source/schema.yml

```diff
@@ -235,9 +235,19 @@ paths:
 
             This is useful to avoid repeating the same analysis multiple times.
             By default this API checks if there are existing analysis related to the md5 in
-            status "running" or "reported_without_fails"
+            status "running" or "reported_without_fails" for analyzer checking.
+
+            However, for status being added in as a factor for Playbooks, 
+            IntelOwl also counts "running" (only if specifically selected), 
+            "reported_with_fails" or "reported_without_fails".
+
+            Also, if a playbook parameter is sent which isn't empty with/without an
+            analyzer, it completely ignores the existence of the analyzer paramter
+            and returns matching playbook jobs with the same observable.
```

Comment:
a more "clean" option could be to invalid the serializer in the case both playbooks and analyzers are set. That would help any integration in terms of understanding this case. Plus it would make sense in my opinion to notify the client that there's something wrong in the provided data.

To do that you could add a `validate` method to that serializer and just do that confrontation

Reply:
Good idea. let me quickly add that.